File: C#/Escuela/Segundo parcial/Menu/Menu.py
# Vamos a hacer un restaurante, escribe el menu


# Información
print(
    "|__________Menu de amburguesas__________|\n|1.Haburguesas.......................$50|\n|2.Pizza.............................$30|\n|3.Agua..............................$50|\n|4.Refresco..........................$15|\n|5.papas.............................$15|)"
)


# Para que seleccionen que van a pedir
print("¿Qué vas a pedir?\n")


while True:
    pedido: int = input("Ingresa el NÚMERO que deseas pedir: ")
    try:
        pedido = int(pedido)
        if pedido > 5:
            raise
        break
    except Exception as e:
        if e == ValueError:
            print("Introduce el dato correctamente")
        else:
            print("Tu numero es superior a 5")

print("Estamos procesando tu pedido")
match pedido:
    case 1:
        print("Se están preparando tus haburguesas")
    case 2:
        print("Se está preparando tu pizza")
    case 3:
        print("Aquí está tu agua")
    case 4:
        print("Aquí está tu refresco")
    case 5:
        print("Aquí están tus papas")

File: C#/Escuela/Segundo parcial/Menu/Menu.cs
using System;
namespace Manu
{
    class Program
    {
        static void Main(string[] args)
        {
            // Información
            Console.WriteLine("|__________Menu de hamburguesas__________|\n|1.Hamburguesas.......................$50|\n|2.Pizza.............................$30|\n|3.Agua..............................$50|\n|4.Refresco..........................$15|\n|5.Papas.............................$15|");

            // Para que seleccionen qué van a pedir
            Console.WriteLine("¿Qué vas a pedir?\n");

            while (true)
            {
                int pedido;
                try
                {
                    pedido = Convert.ToInt32(Console.ReadLine());
                    if (pedido > 5)
                    {
                        throw new Exception();
                    }
                    break;
                }
                catch (Exception e)
                {
                    if (e.GetType() == typeof(FormatException))
                    {
                        Console.WriteLine("Introduce el dato correctamente");
                    }
                    else
                    {
                        Console.WriteLine("Tu número es superior a 5");
                    }
                }
            }

            Console.WriteLine("Estamos procesando tu pedido");

            switch (pedido)
            {
                case 1:
                    Console.WriteLine("Se están preparando tus hamburguesas");
                    break;
                case 2:
                    Console.WriteLine("Se está preparando tu pizza");
                    break;
                case 3:
                    Console.WriteLine("Aquí está tu agua");
                    break;
                case 4:
                    Console.WriteLine("Aquí está tu refresco");
                    break;
                case 5:
                    Console.WriteLine("Aquí están tus papas");
                    break;
                default:
                    break;
            }
        }
    }

}
